src/script/descriptor.cpp

```diff
@@ -897,8 +897,8 @@ std::unique_ptr<DescriptorImpl> ParseScript(uint32_t key_exp_index, Span<const c
             providers.emplace_back(std::move(pk));
             key_exp_index++;
         }
-        if (providers.empty() || providers.size() > 16) {
-            error = strprintf("Cannot have %u keys in multisig; must have between 1 and 16 keys, inclusive", providers.size());
+        if (providers.empty() || providers.size() > MAX_PUBKEYS_PER_MULTISIG) {
+            error = strprintf("Cannot have %u keys in multisig; must have between 1 and %i keys, inclusive", providers.size(), MAX_PUBKEYS_PER_MULTISIG);
```

Comment:
Please use `%d` or `%s` here.

`%i` is explicitly locale-involved, which we don't want in RPC.

Reply:
Done